cypress/integration/browser/EmptyEventsStreams.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Empty event stream
```

Comment:
Additionally the file name could be just EditQuery.feature as empty event stream is not a feature but a state.